CONTRIBUTING.md

```diff
@@ -195,8 +195,8 @@ This key is for boolean properties associated with the test.
 - **noStrict** - only run the test in "sloppy" mode
 - **module** - interpret the source text as
   [module code](https://tc39.github.io/ecma262/#sec-modules)
-- **raw** - execute the test without any modification (no helpers will be
-  available); necessary to test the behavior of directive prologue; implies
+- **raw** - execute the test without any modification (no harness files will be
+  included); necessary to test the behavior of directive prologue; implies
```

Comment:
```suggestion
- **raw** - execute the test without any modification (no harness files or extra content must be
  included); necessary to test the behavior of directive prologue; implies
```
This is a nit pick, but reflects better the correct functionality of the raw flagged files. If they will be included or not, that depends on the runner, which is not controlled by this repo.